pkg/install/artifact/artifact.go

```diff
@@ -267,10 +268,10 @@ func (c *Writer) makeHelmReleaseObjects(artifact profilesv1.Artifact, installati
 	return helmRelease, cfgMap
 }
 
-func (c *Writer) makeHelmRepository(url string, name string, installation profilesv1.ProfileInstallation) *sourcev1.HelmRepository {
+func (c *Writer) makeHelmRepository(url, artifactName string, installation profilesv1.ProfileInstallation) *sourcev1.HelmRepository {
```

Comment:
missing type for url? 🤔

Reply:
this is a golang feature :smile: Rather than:
```
func(foo string, bar string)
```

you can do:
```
func(foo, bar string)
```

Reply:
wow, I had no clue you could do that with go, I learnt something new yay 👍🏻